src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -90,6 +118,11 @@ class ProfileCallTreeContextMenu extends PureComponent {
           >
             Function Name
           </MenuItem>
+          {this.isJS()
```

Comment:
I mean, having the body at the top of `render`, because it's not used anywhere else as far as I see, and `render` being quite short it doesn't impair the readability.